Update batch-docker-container-workloads.md (#115060)
Azure/azure-content: articles/batch/batch-docker-container-workloads.md

  - Offer: `centos-container-rdma`
  - Offer: `ubuntu-server-container-rdma`

- Publisher: `microsoft-dsvm`
  - Offer: `ubuntu-hpc`

#### Notes

>   The docker data root of the above images lie in different places:
>   - for the batch image  `microsoft-azure-batch` (Offer: `centos-container-rdma`, etc), the docker data root is mapped to `/mnt/batch/docker`, which is usually located on the temporary disk.
>   - for the HPC image, or  `microsoft-dsvm` (Offer: `ubuntu-hpc`, etc), the docker data root is unchanged from the Docker default which is `/var/lib/docker` on Linux and `C:\ProgramData\Docker` on windows. This folders are usually located on the OS disk.
> 
>     When using non-Batch images, the OS disk has the potential risk of being filled up quickly as container images are downloaded.

Reviewer: ```suggestion
  The docker data root of the above images lies in different places:
  - For the batch image `microsoft-azure-batch` (Offer: `centos-container-rdma`, etc), the docker data root is mapped to `/mnt/batch/docker`, which is usually located on the temporary disk.
  - For the HPC image, or `microsoft-dsvm` (Offer: `ubuntu-hpc`, etc), the docker data root is unchanged from the Docker default which is `/var/lib/docker` on Linux and `C:\ProgramData\Docker` on Windows. These folders are usually located on the OS disk.

    When using non-Batch images, the OS disk has the potential risk of being filled up quickly as container images are downloaded.
```

>     #### Potential Solutions for Customer
> a) Change the docker data root in a start task when creating a pool in BatchExplorer，example of Start Task command like this:

Reviewer: ```suggestion
#### Potential Solutions for Customer

Change the docker data root in a start task when creating a pool in BatchExplorer. Here's an example of the Start Task command:
```

  - Offer: `centos-container-rdma`
  - Offer: `ubuntu-server-container-rdma`

- Publisher: `microsoft-dsvm`
  - Offer: `ubuntu-hpc`

#### Notes
  The docker data root of the above images lies in different places:
  - For the batch image `microsoft-azure-batch` (Offer: `centos-container-rdma`, etc), the docker data root is mapped to `/mnt/batch/docker`, which is usually located on the temporary disk.
  - For the HPC image, or `microsoft-dsvm` (Offer: `ubuntu-hpc`, etc), the docker data root is unchanged from the Docker default which is `/var/lib/docker` on Linux and `C:\ProgramData\Docker` on Windows. These folders are usually located on the OS disk.

    When using non-Batch images, the OS disk has the potential risk of being filled up quickly as container images are downloaded.

> #### Potential Solutions for Customer

Reviewer: ```suggestion
#### Potential solutions for customer
```
